DEVELOPMENT.md

```diff
@@ -76,3 +76,36 @@ Version bumping policy is as follows:
 Use the `:error_string` metadata key when logging, which is for additional information that we want to log but don't necessarily want searchable or parsed for schema updating.
 
 For example, do `Logger.error("Some error", error_string: inspect(params) )`
+
+## Deployment
+
+### GRPC Server
```

Comment:
can we move this text block to the docs self-hosted.md file instead? this looks like self-hosting setup information. We can have a section called `## GRPC` and a subheader `### Example GRPC Setup on GCP`


For the DEVELOPMENT.md just need to note down the GRPC configs that we are using

Reply:
we can slim down steps to just what our setup requires, though it isn't really development specific. would be more appropriate to go into our deployment notion doc actually.

Reply:
This is what our setup requires 😅 no slimming down of the steps

As for moving it out can I would rather start building a Deployment.md so we move that information out of Notion and closer to the code

Reply:
Yup that sounds good

Reply:
Moved this to DEPLOYMENT.md . The docs in Notion are outdated so we can work on this document instead as we do the changes (e.g. moving to A B deployment)